bigframes/core/explode.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 Google LLC
```

Comment:
Note: Intentionally not 2024 because this code was pulled from a file with date marked as 2023.